lib/AST/ModuleNameLookup.cpp

```diff
@@ -19,9 +19,12 @@ using namespace swift;
 using namespace namelookup;
 
 namespace {
-  using ModuleLookupCache = llvm::SmallDenseMap<ModuleDecl::ImportedModule,
-                                                TinyPtrVector<ValueDecl *>,
-                                                32>;
+  // The usable results of a lookup in a particular module may differ based on
+  // where the lookup is happening.
```

Comment:
It's not immediately apparent what you are thinking of,  because the "where" could be more specific. Or maybe it's the "usable"? Do you any desire to expound a bit? Maybe with an example?